src/com/epam/homework/university/models/Faculty.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.homework.university.models;
+
+import com.epam.homework.university.Helper;
+
+import java.text.DecimalFormat;
+
+public class Faculty {
+    private String name;
+    private Group[] groups;
+
+    public String getName() {
+        return name;
+    }
+
+    public Group[] getGroups() {
+        return groups;
+    }
+
+    public Faculty(String name) {
+        this.name = name;
+    }
+
+    public Faculty(String name, Group[] groups) {
+        this.name = name;
+        if (groups == null || groups.length == 0){
```

Comment:
You may write some utility method which accepts an array and checks if it is null or empty not to repeat that check everywhere

Reply:
Added validateArray method to to Helper class